taichi/rhi/device.h

```diff
@@ -10,6 +10,8 @@
 #include "taichi/rhi/device_capability.h"
 #include "taichi/rhi/arch.h"
 
+#include "taichi/program/kernel_profiler.h"
```

Comment:
`device.h` should not include any taichi headers

Reply:
Fixed it :)